ui/src/components/workflow/executions/WorkflowList.js

```diff
@@ -83,7 +83,11 @@ const Workflow = React.createClass({
       selectedWFEs:[],
       bulkProcessOperation: "",
       bulkProcessInFlight: false,
-      bulkProcessSuccess: false
+      bulkProcessSuccess: false,
+      error:false,
+      bulkError:false,
+      bulkValidationMessage:"",
```

Comment:
can we expand this? On my side I don't have 1 generic bulk error, Each failed workflow will have one so I basically have 2 lists, one with successfully processed id's and another one, with failed ids and corresponding error messages, ie {
    "bulkSuccessResults": [
        "16cb26da-87c4-4912-a8fc-79eacc0ed4cc",
        "16cb26da-87c4-4912-a8fc-79eacc0ed4c2"
    ],
    "bulkErrorResults": [
        {
            "16cb26da-87c4-4912-a8fc-79eacc0ed4c2": "No such workflow found by id: 16cb26da-87c4-4912-a8fc-79eacc0ed4c2"
        }
    ]
}